### PR TITLE
Add VCH uuid into image storage usage calculation

### LINKED ISSUE #8414: docker info calculated wrong storage quota when 2 VCH are using the same image-store folder
Using vic1.5.0, we deploy 2 VCH which are using the same image folder through parameter --image-store sharedVmfs-0/VCH_Images.
A few container-vms are created under VCH_1 and no container-vms are created under VCH_2. When using docker -H ** info cmd, we can see that the output shows the wrong storage usage and images storage usage for VCH_2.
**pwd
/vmfs/volumes/sharedVmfs-0/VCH_Images/VIC
ls
422771d9-9e61-2864-d3f2-dd894f13ea92  42277c18-ae49-9c64-4f27-48320c27eb19**

### PR DESCRIPTION
**Pull request by wjun**
Fixes #8414 
[specific ci=Group26-Storage-Quota]

### CODE REVIEW

File: lib/apiservers/portlayer/restapi/handlers/storage_handlers.go [hunk 1]
```diff
@@ -733,12 +733,12 @@ func (h *StorageHandlersImpl) StatPath(params storage.StatPathParams) middleware
...
 }
 
-// ListImages returns a list of images in a store
+// GetImageStorageUsage returns images storage usage for this VCH
 func (h *StorageHandlersImpl) GetImageStorageUsage(params storage.GetImageStorageUsageParams) middleware.Responder {
 	op := trace.NewOperationFromID(context.Background(), params.OpID, "GetImageStorageUsage(%s)", params.StoreName)
 	defer trace.End(trace.Begin("GetImageStorageUsage", op))
 
-	result, err := h.imageCache.DataStore.GetImageStorageUsage(op)
+	result, err := h.imageCache.DataStore.GetImageStorageUsage(op, params.StoreName)
```
Comment (DanielXiao) [inline lib/apiservers/portlayer/restapi/handlers/storage_handlers.go]: Does StoreName equal VCH UUID? I think it is the VCH name.

File: lib/apiservers/portlayer/restapi/handlers/storage_handlers.go [hunk 2]
```diff
@@ -733,12 +733,12 @@ func (h *StorageHandlersImpl) StatPath(params storage.StatPathParams) middleware
 
 }
 
-// ListImages returns a list of images in a store
+// GetImageStorageUsage returns images storage usage for this VCH
```
Comment (DanielXiao) [inline lib/apiservers/portlayer/restapi/handlers/storage_handlers.go]: Does calculation distinguish volume stores from differen VCH?

Comment (DanielXiao) [reply lib/apiservers/portlayer/restapi/handlers/storage_handlers.go]: What if I specify --volume-store sharedVmfs-0/<VCH name>:default for better folder struction?

File: lib/apiservers/portlayer/restapi/handlers/storage_handlers.go (hunk 1 above)
Comment (wjun) [reply lib/apiservers/portlayer/restapi/handlers/storage_handlers.go]: Yes, it is VCH UUID

File: lib/apiservers/portlayer/restapi/handlers/storage_handlers.go (hunk 2 above)
Comment (wjun) [reply lib/apiservers/portlayer/restapi/handlers/storage_handlers.go]: Good catch. Fixed.

**Pull request merged**